### PR TITLE
Group reporting guidelines guidelines in ToC

### PR DESCRIPTION
**Pull request by krassowski**
### Background or context to help others understand the change.

I was surprised to see that reporting during events was in the Jupyter Day section. I think it fits better next to the online reporting section.

The current structure was introduced in https://github.com/jupyter/governance/pull/115. @choldgraf do you have an opinion on this one?

### A brief summary of the change.

| Before | After |
|---|---|
|Screenshot from 2023-04-12 06-29-42| Screenshot from 2023-04-12 06-27-12 |

### What is the reason for this change?

Conceptually, there are in-person Jupyter events other than JupyterDay.

### Alternatives to making this change and other considerations.

Leave it as is, since it might be indeed of interest of JupyterDay attendees?

> ## The process ❗
> 
> The process for changing the governance pages is as follows:
> 
> * Open a pull request **in draft state**. This triggers a discussion and iteration phase
>   for your proposed changes.
> * When you believe enough discussion has happened,
>   **move the pull request to an active state**. This triggers a vote.
> * During the voting phase, no substantive changes may be made to the pull request.
> * The Executive Council and Software Steering Council will vote, and at the end of voting the pull request is merged or closed.
> 
> The discussion phase is meant to gather input and multiple perspectives from the community.
> Make sure that the community has had an opportunity to weigh in on
> the change **before calling a vote**. A good rule of thumb is to ask several Council
> members if they believe that it is time for a vote, and to let at least one person review
> the pull request for structural quality and typos.

### CODE REVIEW

**Comment by choldgraf** [comment]
It makes sense to me 👍

**Comment by JohanMabille** [comment]
+1 for the change

**Review by Carreau — APPROVED** [review]

**Comment by krassowski** [comment]
Taking out of draft as there appears to be a consensus from initial replies.

**Comment by choldgraf** [comment]
In my opinion, this PR doesn't need to have consensus from the Executive Committee or the SSC. It is making information more discoverable and intuitively-ordered, not changing its meaning or any policy. So my suggestion is that somebody with merge rights (I do not have them) just merge this, unless there are any objections.

**Pull request merged**
